source/hands_on/td3_bc.rst

```diff
@@ -0,0 +1,112 @@
+|                     |                 |                                                     |github.com/opendilab/     | d3rlpy(3690)         |
+|Hopper               |                 |                                                     |DI-engine/blob/main/dizoo/|                      |
+|                     |  3653           |.. image:: images/benchmark/hopper_td3bc.png         |d4rl/config/hopper_sac_   |                      |
+|(Medium Expert)      |                 |                                                     |td3bc_medium_expert       |                      |
+|                     |                 |                                                     |_config.py>`_             |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+
+
++---------------------+-----------------+----------------+---------------+----------+----------+
+| environment         |random           |medium replay   |medium expert  |medium    |expert    |
++=====================+=================+================+===============+==========+==========+
+|                     |                 |                |               |          |          |
+|Halfcheetah          |1592             |5192            |13037          |5257      |13247     |
+|                     |                 |                |               |          |          |
++---------------------+-----------------+----------------+---------------+----------+----------+
+|                     |                 |                |               |          |          |
+|Walker2d             |345              |1724            |3653           |3268      |3664      |
+|                     |                 |                |               |          |          |
++---------------------+-----------------+----------------+---------------+----------+----------+
+|                     |                 |                |               |          |          |
+|Hopper               |985              |2317            |5066           |3826      |5232      |
+|                     |                 |                |               |          |          |
++---------------------+-----------------+----------------+---------------+----------+----------+
+
+**Note**: the D4RL environment used in this benchmark can be found `here <https://github.com/rail-berkeley/d4rl>`_.
+
+Other Public Implementations
+----------------------------
+
+- `Official implementation`_
+- d3rlpy_
+
+.. _`Official implementation`: https://github.com/sfujim/TD3_BC
+.. _d3rlpy: https://github.com/takuseno/d3rlpy
+
+References
+-----------
+Scott Fujimoto, Shixiang Shane Gu: “A Minimalist Approach to Offline Reinforcement Learning”, 2021; [https://arxiv.org/abs/2106.06860 arXiv:2106.06860].
```

Comment:
swap `references` and `Other Public Implementations`

Reply:
closed in latest commit